config/default.yaml

```diff
@@ -539,6 +539,9 @@ import:
       # We recommend to only enable magnet URI/torrent import if you trust your users
       # See https://docs.joinpeertube.org/maintain-configuration?id=security for more information
       enabled: false
+  synchronization:
```

Comment:
`video-channel-synchronization`

We can imagine in the future also sync videos, to automatically update their title etc :)